build.sbt

```diff
@@ -3137,6 +3138,7 @@ lazy val `library-manager-test` = project
     )
   )
   .dependsOn(`library-manager`)
+  .dependsOn(`process-utils`)
```

Comment:
Shouldn't this be just a `Test` dependency?

Reply:
No, this is correct.